### PR TITLE
Controls add key to previous pano

### PR DESCRIPTION
**Pull request by gliegard**
## Description
Immersive: Add the possibility to go backward with key S

- Street controls : Add key binding to previsous pano
- Oriented Image Layer : 
  - Add the previsous pano position in the callback
  - Also, manage case wise no background object
- Immersive example updated

### CODE REVIEW

**Review by zarov — APPROVED** [review]
Nice thanks, however I would like to discuss one thing before merging: the choice of the keys. In your case, you have a AZERTY keyboard, so it makes sense to have `Z`, `A` etc... as keys to move inside the view. But with another keyboard, it is quite weird to use those keys. That's why I'm proposing more logical keys to use:
- `P` for previous
- `N` for next
- `R` (for reset) or `C` (for current) for the current position

WDYT ?

**Comment by gliegard** [comment]
I can add a setter with key code to let the user remap the keys ?
like
setGoForwardKeyCode(keyCode)
set GoBackwardKeyCode(keyCode)

WDYT ?

**Comment by zarov** [comment]
Sounds good to me, thanks.

**Comment by gliegard** [comment]
ok, updated :)

File: src/Controls/StreetControls.js [hunk 1]
```diff
@@ -51,8 +51,16 @@ class StreetControls extends FirstPersonControls {
         this.player.addEventListener('animation-frame', this.updateView.bind(this));
 
         // twos positions used by this control : current and next
+        this.previousPosition = undefined;
         this.currentPosition = undefined;
         this.nextPosition = undefined;
+
+        // store key binding to go to next position, default to Z
```
Comment (zarov) [inline src/Controls/StreetControls.js]: Please move those comments before the class declaration, as `@property` of the class.

Comment (gliegard) [reply src/Controls/StreetControls.js]: ok, done

Comment (zarov) [reply src/Controls/StreetControls.js]: Thanks, merged

Comment (gliegard) [reply src/Controls/StreetControls.js]: Yeahh , thanks :) :)

**Pull request merged**